packages/MarkdownEditor-Tests.package/MarkdownDelimiterEmphasisTest.class/instance/testEmphasisAttributeBold2.st

```diff
@@ -0,0 +1,17 @@
+tests - attributes
+testEmphasisAttributeBold2
```

Comment:
maybe 'testDifferentDelimitersCreateBoldAttribute' and then rename the above test case to 'testSameDelimitersCreateBoldAttribute'